### PR TITLE
[Release Guide] Document Sprints

### PR DESCRIPTION
**Pull request by mkolesnik**
Depends on #711 

Signed-off-by: Mike Kolesnik <mkolesni@redhat.com>

### CODE REVIEW

File: src/content/development/contribution-guide/_index.en.md [hunk 1]
```diff
@@ -50,7 +50,28 @@ To be expanded
 
 ### Sprints
 
-To be expanded
+Sprints are 3 week periods which encapsulate work on the release.
+Most sprints focus on active development of the current version, while some focus on additional aspects such as design and stabilization.
+Specific sprints and their contents are details in the following sections.
```
Comment (skitt) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
Specific sprints and their contents are detailed in the following sections.
```

File: src/content/development/contribution-guide/_index.en.md [hunk 2]
```diff
@@ -50,7 +50,28 @@ To be expanded
...
+
+Each sprint starts and ends on a boundry day, when one sprint ends another one begins.
```
Comment (skitt) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
Each sprint starts and ends on a boundary day, when one sprint ends another one begins.
```

File: src/content/development/contribution-guide/_index.en.md [hunk 3]
```diff
@@ -50,7 +50,28 @@ To be expanded
...
+The boundry day is **Monday**.
```
Comment (skitt) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
The boundary day is **Monday**.
```

File: src/content/development/contribution-guide/_index.en.md [hunk 4]
```diff
@@ -50,7 +50,28 @@ To be expanded
...
+
+On the sprint boundry day we will:
```
Comment (skitt) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
On the sprint boundary day we will:
```

File: src/content/development/contribution-guide/_index.en.md [hunk 5]
```diff
@@ -50,7 +50,28 @@ To be expanded
...
+
+* Perform a milestone release (When applicable).
```
Comment (skitt) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
* Perform a milestone release (when applicable).
```

**Review by skitt — CHANGES_REQUESTED** [review]

**Comment by mkolesnik** [comment]
Updated according to fixes and also added a paragraph about test days.

Make sure to squash when merging!

**Review by skitt — APPROVED** [review]

**Review by dfarrell07 — APPROVED** [review]

File: src/content/development/contribution-guide/_index.en.md [hunk 6]
```diff
@@ -50,7 +50,33 @@ To be expanded
 
 ### Sprints
 
-To be expanded
+Sprints are 3 week periods which encapsulate work on the release.
+Most sprints focus on active development of the current version, while some focus on additional aspects such as design and stabilization.
+Specific sprints and their contents are detailed in the following sections.
+
+Most sprints will end with a milestone pre-release, the designation of which is detailed in the following sections.
```
Comment (tpantelis) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
Most sprints will end with a milestone pre-release as detailed in the following sections.
```

File: src/content/development/contribution-guide/_index.en.md [hunk 7]
```diff
@@ -50,7 +50,33 @@ To be expanded
...
+This allows the community and project members to verify the project is stable and test any fixes or features that were added during the
+sprint.
+For most sprints a [test day](#test-days) to test the pre-release will be held after its creation.
```
Comment (tpantelis) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
A formal [test day](#test-days) may be held to facilitate testing of the pre-release.
```

File: src/content/development/contribution-guide/_index.en.md [hunk 8]
```diff
@@ -50,7 +50,33 @@ To be expanded
...
+
+Each sprint starts and ends on a boundary day, when one sprint ends another one begins.
```
Comment (tpantelis) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
Each sprint ends on a boundary day which also marks the beginning of the next sprint.
```

File: src/content/development/contribution-guide/_index.en.md [hunk 9]
```diff
@@ -50,7 +50,33 @@ To be expanded
...
+The boundary day is **Monday**.
```
Comment (tpantelis) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
The boundary days occur on a **Monday**.
```

**Review by tpantelis — CHANGES_REQUESTED** [review]

**Comment by mkolesnik** [comment]
Thanks for the suggestions, please make sure to squash before merging!

**Review by tpantelis — APPROVED** [review]

**Pull request merged**